rust/agama-cli/src/config.rs

```diff
@@ -193,24 +193,19 @@ fn validate_local(url_or_path: CliInput, insecure: bool) -> anyhow::Result<Valid
     }
 }
 
-async fn validate_silently(
+async fn validate(
     client: &BaseHTTPClient,
     url_or_path: CliInput,
+    silent: bool,
```

Comment:
To be honest, I do not like that `validate()` takes care of printing anything. And adding a `silent` argument does not make it any better.

However, let's consider that for a future refactoring during the `api-v2` adoption.